index.html

```diff
@@ -950,9 +950,40 @@ <h4>
     </pre>
   </section>
 
+  <section><h3>General information about writing data</h3>
+    <p>
+      Writing data is generally straight-forward, but there are a few
+      gotcha's around how writing works with NFC.
+    </p>
+    <p>
+      An NFC reader works by polling, so in order to be able to
+      write to a tag, a tag first needs to be found and read, which
+      means that polling needs to be initialized first.
+    </p>
+    <p>
+      If polling is not initiated already by first calling `scan()`,
+      then the `write()` method will initiate it temporarily until a
+      tag was found and read, and writing to it was attempted.
+    </p>
+    <p>
+      This means that the flow is that first a read it performed
+      as the tag is first found, then followed by a write.
+    </p>
+    <p>
+      This means that if `scan()` is running and you have an event
+      listener for the `reading` event, it will be dispatched once
+      during a `write()` operation, which might not be the intended
+      behavior.
+    </p>
+    <p>
+      In the following sections we will discuss how you can easily
+      deal with this behavior, but first a few simple examples.
+    </p>
+  </section>
+
   <section><h3>Write a text string</h3>
     <p>
-      Writing a text string to an NFC tag is straightforward.
+      Writing a text string to an NFC tag is straight-forward.
```

Comment:
straightforward was fine
https://www.merriam-webster.com/dictionary/straightforward

Reply:
Then I need to fix in more places